mqtt/mqtt-broker/src/transport.rs

```diff
@@ -79,10 +41,17 @@ impl Transport {
         Ok(Transport::Tcp(tcp))
     }
 
-    async fn new_tls<A>(addr: A, identity: Identity) -> Result<Self, InitializeBrokerError>
+    pub async fn new_tls<A>(addr: A, cert_path: PathBuf) -> Result<Self, InitializeBrokerError>
     where
         A: ToSocketAddrs + Display,
     {
+        info!("Loading identity from {}", cert_path.display());
```

Comment:
This is not identity cert, right?

Reply:
Actually it is. native_tls works only with pkcs12 identity certs